### ISSUE TITLE
Timezone issue in AnalyzerOptions

### ISSUE THREAD

**Issue by akamra** [google/zetasql#7]
Hiya,

Thanks for releasing this publicly! I've been digging through the codebase for the past few weeks. My colleague  and I noticed this small "bug" in analyzer.cc, when initializing analyzerOptions.

`CHECK(LoadTimeZone("America/Los_Angeles", &default_timezone_));`

**Comment by matthewcbrown** [comment]
Can you clarify what the bug is? That line is initializing the default timezone to "America/Los_Angeles" - (which is a bit arbitrary, but does not seem incorrect exactly), this can to overwritten with AnalyzerOptions::set_default_time_zone if desired.

**Comment by akamra** [comment]
Yes, you are right. It's not actually a bug. I figured it was something that might be left over from testing. The issue then becomes that all of the tests fail, which confused us quite a bit when we first got everything up and running. It was easier for us to delete the check, then find all the occurrences when you initialized analyzerOptions in your tests.

**Comment by matthewcbrown** [comment]
Odd. This could be some machine configuration/build environment issue, we don't see those test failures. Can you describe what platform you are building on?

**Comment by matthewcbrown** [comment]
can you try installing 'tzdata' package (assuming you're on linux)?
